charts/s1-agent/values.yaml

```diff
@@ -201,6 +201,22 @@ agent:
   apparmorLocalhostProfileName: "" # Set to a custom AppArmor profile name if type is Localhost, otherwise leave empty
   seccompProfile: # works for k8s version >= 1.19, older application was via annotation
     type: Unconfined
+  startupProbe:
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 60
```

Comment:
60 seconds is not too long? in case of a failure it can make the startup cycle time at least 150 seconds